python/cudf/cudf/core/column/string.py

```diff
@@ -5464,6 +5464,7 @@ class StringColumn(column.ColumnBase):
 
     def __init__(
         self,
+        data: Union[Buffer, None] = None,
```

Comment:
Prefer `Optional[Buffer]` as the type here.

Reply:
done.